daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Typing.scala

```diff
@@ -456,7 +456,11 @@ private[validation] object Typing {
       iface match {
         case DefInterface(requires, param, fixedChoices, methods, precond) =>
           val env = introExprVar(param, TTyCon(ifaceName))
-          requires.foreach(required => handleLookup(ctx, interface.lookupInterface(required)))
+          for {
+            required <- requires
+            requiredRequired <- handleLookup(ctx, interface.lookupInterface(required)).requires
+            if !requires(requiredRequired)
+          } throw EMissingRequiredInterface(ctx, ifaceName, required, requiredRequired)
```

Comment:
should this throw the new error, `ENotClosedinterfaceRequires`?

Reply:
good catch.